app/lib/services/voip/CallLifecycle.test.ts

```
@@ -40,13 +40,29 @@ jest.mock('../../../containers/ActionSheet', () => ({
+jest.mock('./MediaSessionInstance', () => ({
+	mediaSessionInstance: {
+		getMediaCall: (...args: unknown[]) => mockGetMediaCall(...args)
+	}
+}));
+
+// Mock getDMSubscriptionByUsername — CallLifecycle.resolveRoomIdFromContact uses it.
+jest.mock('../../database/services/Subscription', () => ({
+	getDMSubscriptionByUsername: jest.fn()
+}));
+
 import type { IClientMediaCall } from '@rocket.chat/media-signaling';
```

ESLint

[error] Import in body of module; reorder to top.

(import/first)